ci/docker/runtime_functions.sh

```diff
@@ -855,7 +830,6 @@ build_ubuntu_gpu_cmake_mkldnn() {
         -DUSE_CUDA=1                            \
         -DUSE_CUDNN=1                           \
         -DUSE_TVM_OP=1                          \
```

Comment:
I think we should turn off this by default and create a separate to test TVM_ON instead.